creator-node/src/migrationManager.js

```diff
@@ -3,7 +3,7 @@ const path = require('path')
 
 const { sequelize } = require('./models')
 
-function runMigrations () {
+async function runMigrations () {
```

Comment:
this is not related at all. i honestly dont know how this is working now without the async here